packages/calypso-e2e/src/lib/pages/plans-page.ts

```diff
@@ -20,6 +20,12 @@ const selectors = {
 		}
 		return `button.is-${ name.toLowerCase() }-plan:visible`;
 	},
+	selectModalUpsellPlanButton: ( name: 'Free' | 'Personal' ) => {
+		if ( name === 'Free' ) {
+			return `button.is-upsell-modal-free-plan:visible`;
+		}
+		return `button.is-upsell-modal-${ name.toLowerCase() }-plan:visible`;
+	},
```

Comment:
**issue:blocking** - CSS locators are now discouraged in favor of accessible locators. You can read more about it [here](https://playwright.dev/docs/locators).